Dockerfile

```diff
@@ -1,6 +1,9 @@
 FROM node:lts-alpine
 
 RUN set -ex && mkdir /app
+RUN apk add --no-cache python3 && ln -sf python3 /usr/bin/python
```

Comment:
The symlink may be not needed if we install those with `apk`.

Reply:
If we install Python 3 by ```apk```, ```/bin/sh: python: not found``` will appear if the yt-dlp and youtube-dl bins are downloaded by wget.

Reply:
You are right.

Reply:
yt-dlp may execute, but youtube-dl (from wget) would have error ```env: can't execute 'python': No such file or directory``` as not like yt-dlp, it requires 'python' instead of 'python3'

Reply:
As youtube-dl is provided by apk, I would remove the linking process.